src/ufuncs.c

```diff
@@ -399,6 +399,29 @@ static void* polygons_without_holes_data[1] = {GEOSLinearRingToPolygon};
 static void* build_area_data[1] = {GEOSBuildArea_r};
 static void* make_valid_data[1] = {GEOSMakeValid_r};
 static void* coverage_union_data[1] = {GEOSCoverageUnion_r};
+static void* GEOSMinimumBoundingCircleWithReturn(void* context, void* geom) {
+  GEOSGeometry* center = NULL;
+  double radius;
+  GEOSGeometry* ret = GEOSMinimumBoundingCircle_r(context, geom, &radius, &center);
+  if (ret == NULL) {
+    return NULL;
+  }
+  GEOSGeom_destroy_r(context, center);
+  return ret;
+}
+static void* minimum_bounding_circle_data[1] = {GEOSMinimumBoundingCircleWithReturn};
+static int GEOSMinimumBoundingRadius(void* context, GEOSGeometry* geom, double* radius) {
```

Comment:
Can you move this to where the other Y_d functions are defined? (this is the Y_Y section)